salt/modules/win_file.py

```diff
@@ -1415,8 +1415,9 @@ def mkdir(
                 salt.utils.win_dacl.set_owner(obj_name=path, principal=owner)
 
             # Set permissions
-            set_perms(
-                path=path,
+            salt.utils.win_dacl.set_perms(
```

Comment:
`__utils__`?

Reply:
Removed all imports that could use __utils__